docs/usage-examples.txt

```diff
@@ -0,0 +1,67 @@
+.. _laravel-usage-examples:
+
+==============
+Usage Examples
+==============
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta::
+   :keywords: set up, runnable, code example
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+Usage examples show runnable code examples to demonstrate frequently used MongoDB
+operations. Each usage example includes the following components:
+
+- Explanation of the MongoDB operation
+- Example controller code that runs the MongoDB operation
+- Example view code that renders the result
+- Output displayed by the view
+
+How to Run the Usage Examples
+-----------------------------
+
+The usage examples are designed to run operations on a MongoDB deployment that contains
+the MongoDB Atlas sample datasets. When you run the example code without this sample data,
+the output might not match. 
+
+You can run the usage examples from the Laravel web application and MongoDB Atlas cluster that you
+set up in the :ref:`laravel-quick-start` guide. After completing the Quick Start, ensure that your
+application meets the following requirements:
+
+- Connects to the ``movies`` collection in the Atlas sample datasets
+- Contains the ``MovieController.php`` and ``browse_movies.blade.php`` files
```

Comment:
Suggestion:

I think it could be easier to verify the requirements as Laravel web app components rather than existence of the collection and files. I think it's also missing mention of the API controller (used for storing data - see https://www.mongodb.com/docs/drivers/php/laravel-mongodb/current/quick-start/write-data/). E.g.

"... application contains the following components:"

- Configuration to use an Atlas cluster that contains the sample datasets
- ``Movie`` model configured to use the MongoDB database
- ``Movie`` view that lists the results
- HTTP and API controller endpoints for displaying and storing ``Movie`` data
"